packages/doctype-caip10-link/src/caip10-link-doctype.ts

```diff
@@ -36,80 +22,107 @@ const throwReadOnlyError = (): Promise<void> => {
 @DoctypeStatic<DoctypeConstructor<Caip10LinkDoctype>>()
 export class Caip10LinkDoctype extends Doctype {
```

Comment:
Did we want to call this class just `Caip10Link`, or is that a future PR?

Reply:
Future PR, under https://github.com/ceramicnetwork/js-ceramic/issues/1115